docs/source/topic-guides/models.md

```diff
@@ -141,17 +141,17 @@ If there is only one _new owner_, the condition will be a simple signature condi
         },
         "uri": "<string>"
     },
-    "new_owners": ["<new owner public key>"]
+    "owners_after": ["<public key of the owner after the transaction happens>"]
 }
 ```
 
 - **Condition header**:
     - `cid`: Condition index so that we can reference this output as an input to another transaction. It also matches
     the input `fid`, making this the condition to fulfill in order to spend the asset used as input with `fid`.
-    - `new_owners`: A list containing one item: the public key of the new owner.
+    - `owners_after`: A list containing one item: the public key of the owner after the transaction happens.
```

Comment:
As per my last comment, and to be consistent with the "new owner" terminology of this subsection, this should probably be:

    - `owners_after`: A list containing one item: the public key of the new owner.